observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator by using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you would want to identify in your flows and metrics. This enhancement can also be used to better identify traffic from or to IPs that are external to the cluster. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
```

Comment:
```suggestion
You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you want to identify in your flows and metrics. This enhancement can also be used to more easily identify traffic from or to IPs that are external to the cluster. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
```
I don't know what "better" means. Is there something specific that was added to the log data that should be called out? Or can we combine the labels and logs for this purpose?

Reply:
If we need more details: this allows to more easily identify external traffic, by using the new `SrcSubnetLabel` and `DstSubnetLabel` (which exist in flow logs and in metrics), which are empty in case of external traffic. idk if we want to call out that somehow or if that's too much details in the release note. This is mentioned in the CRD doc and in the documented sample metrics.

Reply:
The extra details only add a sentence and a phrase. Not bad and like you said, it kinda primes the sample metrics that are documented. I will add it.